src/Sulu/Bundle/MediaBundle/Collection/Manager/CollectionManagerInterface.php

```diff
@@ -121,7 +121,7 @@ public function save(array $data, ?int $userId = null, bool $breadcrumb = false)
      *
      * @param int $id the id of the category to delete
      */
-    public function delete($id);
+    public function delete($id, bool $forceRemoveChildren = false);
```

Comment:
Should not be added in the interface we should handle this over `func_num_args` and `func_get_arg` like in the other methods for bc reasons.

Reply:
We also adjusted some interfaces for sulu 2.3 and agreed, that it's okay to make such a bc break in a minor version. Why should we handle this differently now?

Reply:
We avoid bc breaks where it is possible and this is easy possible.